pkg/ddc/goosefs/shutdown_test.go

```diff
@@ -358,7 +357,7 @@ func TestGooseFSEngineCleanAll(t *testing.T) {
 	type fields struct {
 		name        string
 		namespace   string
-		cm          *corev1.ConfigMap
+		cm          *v1.ConfigMap
```

Comment:
To keep the consistence of code style and make the code easy to read, I suggest  using `corev1` to replace `v1`. Thanks.

Reply:
SGTM